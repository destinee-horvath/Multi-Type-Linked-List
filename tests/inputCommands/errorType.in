TYPE
type
TYPE -3
TYPE abc
