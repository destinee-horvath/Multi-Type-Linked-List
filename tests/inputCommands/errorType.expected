INVALID COMMAND: TYPE
INVALID COMMAND: INPUT
INVALID COMMAND: TYPE
INVALID COMMAND: TYPE
